src/modules/queue/QueueComponent.jsx

```diff
@@ -103,50 +82,14 @@ const QueueButton = styled.button`
 	border-radius: 50%;
 	box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.1);
 	cursor: pointer;
-	display: inline-block;
-	height: 64px;
-	margin-top: 16px;
+	height: 4rem;
+	width: 4rem;
+	margin-top: 1rem;
 	outline: none;
-	position: relative;
-	width: 64px;
-	z-index: 2;
-
-	image {
-		height: 20px;
-		width: 20px;
-		opacity: 1;
-		transform: transition(all 0.1s);
-	}
 
-	.queue-x {
```

Comment:
Very impressive that you created an X out of pure CSS rules. I would have never thought to do that - that's some serious CSS-Fu :fire: :tophat: 

But it's ultimately unneccessary. You have a bunch of complicated styling to handle one of them being an image and the other being HTML. Just make them both images and style them identically.